### DISCUSSION TITLE
Feature Discussion

### DISCUSSION THREAD

**Discussion by MauPatel2023** [MauPatel2023/Tafe230214#1] [General]
What colour should the background be ?

**Comment by shruthi-rajasekaran** [comment]
Blue

**Comment by shruthi-rajasekaran** [comment]
Hey Mau, I'm working on the Mortgage calculator. Can you please work on the Currency calculator?

**Comment by MauPatel2023** [comment]
Sure no problem, what features should we include in the currency calculator ?

**Comment by shruthi-rajasekaran** [comment]
You need to create a Foreign exchange calculator capturing the user input and display output.

**Comment by MauPatel2023** [comment]
I have coded the currency calculator on VS, Can we merge it with the development branch?

**Comment by shruthi-rajasekaran** [comment]
I am still working on my code for Mortgage Calculator. I will complete it and push the code to the development branch.

**Comment by shruthi-rajasekaran** [comment]
Can you please let me know once you have merged the code into the development branch so we can start working on the next part of the assignment.

**Comment by MauPatel2023** [comment]
Sure, no worries.
